lib/src/Uncertainty/Distribution/HistogramFactory.cxx

```diff
@@ -133,6 +140,63 @@ Histogram HistogramFactory::buildAsHistogram(const Sample & sample,
   return result;
 }
 
+Histogram HistogramFactory::buildAsHistogram(const Sample & sample,
+    const Scalar first,
+    const Point width) const
+{
+  const UnsignedInteger size = sample.getSize();
+  if (size == 0) throw InvalidArgumentException(HERE) << "Error: cannot build an Histogram based on an empty sample.";
+  if (sample.getDimension() != 1) throw InvalidArgumentException(HERE) << "Error: can build an Histogram only if dimension equals 1, here dimension=" << sample.getDimension();
+  const UnsignedInteger binNumber = width.getSize();
+  if (binNumber == 0) throw InvalidArgumentException(HERE) << "Error: expected a positive number of bin, got 0.";
+  // Construct the histogram
+  Point heights(binNumber, 0.0);
```

Comment:
No need to initialize the Point to zero as it is done by default.

Reply:
OK.